uploadNeuroDB/bin/mincpik

```diff
@@ -1,4 +1,156 @@
+values when generating the images (see the manpage for C<minclookup>).
+
+=item *
+B<-range min max> : specifies  the  range  of voxel values to consider, in their
+integer representation. Default is the full range for the type and sign. 
+This option is ignored for floating point values. See manpage for C<mincreshape>.
+
+=item *
+B<-image_range min max> : Normalize the image to a given C<min> and C<max> real value (not
+voxel value). Cannot be used with C<-auto_range>. See manpage for C<mincreshape>.
+
+=item *
+B<-auto_range> : Automatically find the range used to normalize the image. This is
+done by finding the 5% to 95% PcT image range using C<mincstats>. This option
+cannot be used with C<-image_range>. If not specified (and if C<-image_range> is not specified) 
+then the entire image (range = 0-100) is used.
+
+=item *
+B<-transverse> : add the transverse slices to the set of slices to consider during
+image generation. The transverse slices are always considered during image generation 
+so this option is useless/obsolete 
+
+=item *
+B<-coronal> : add the coronal slices to the set of slices to consider during image generation
+
+=item *
+B<-sagittal> : add the sagittal slices to the set of slices to consider during image generation
+
+=item *
+B<-axial> : same as transverse
+
+=item *
+B<-allthree> : deprecated. Use C<-triplanar> instead.
+
+=item *
+B<-triplanar> : use slices in all dimensions to generate the image. Same as specifying
+C<'-axial -coronal -sagittal'>
+
+=item *
+B<-tilesize size> : the size in pixel of each image when C<-triplanar> is used.
+
+=item *
+B<-sagittal_offset offset> : Offset the sagittal slices by C<offset>. For example:
+
+C<./mincpik -slice 15 -sagittal -sagittal_offset 3 in.mnc out.png>
+
+would generate an image using slice at index 18 (15+3). Cannot be used 
+with C<-sagittal_offset_perc> (see below)
+
+=item *
+B<-sagittal_offset_perc perc> : same as C<-sagittal_offset> but with an offset given as
+a percentage of the total number of slices in the sagittal dimension. Note that
+C<perc> should be a number between 0 and 100. This option cannot be used in conjunction
+with C<-sagittal_offset>
+
+=item *
+B<-vertical> : when generating triplanar images, put the image associated to each dimension
+stacked in a vertical row. This is the default behaviour when C<-triplanar> is used.
+
+=item *
+B<-horizontal> : when generating triplanar images, put the image associated to each
+dimension stacked in a horizontal row.
+
+=back
+
+The F<outfile> command line argument is optional. If not specified, output will be a PNG image
+written to STDOUT. If F<outfile> is specified, the file extension will determine the type of
+the generated image (F<.jpg> for JPEG images, F<.gif> for GIF images, etc...)
+
+=head1 TO DO
+
+Nothing
+
+=head1 BUGS
+
+None reported.
+
+=head1 LICENSING
+
+License: GPLv3
+
+=head1 AUTHORS
+
+Andrew Janke - a.janke@gmail.com
```

Comment:
Add a comma after the first author, so that when displayed in one row there is a separation.

Reply:
Ok